docs/developer-guide/nodejs-api.md

```diff
@@ -224,7 +224,7 @@ The return value is an object containing the results of the linting operation. H
             output: "foo;",
             messages: [
                 {
-                    fatal: false,
+                    fatal: false, // Note that this is _only_ included if parsing failed, otherwise it's `undefined`
```

Comment:
Sorry for nitpicking, but I'm afraid people might be confused about the reference to "undefined" (i.e., they might assume the key should be present and valued as undefined). Could we just say "Note that this is only included if parsing failed"?